apps/vc-api/test/vc-api/workflows/resident-card/resident-card.e2e-suite.ts

```diff
@@ -94,9 +95,16 @@ export const residentCardWorkflowSuite = () => {
       didAuthStepId
     );
 
-    // As the issuer, check the result of the transaction verification
-    expect(stepSubmission.presentationSubmission.verificationResult.verified).toBeTruthy();
-    expect(stepSubmission.presentationSubmission.verificationResult.errors).toHaveLength(0);
+    // As the issuer, check the result of the step verification
+
+    // TODO: revisit the isComplete test
+    // expect(stepSubmission.step.isComplete).toBeTruthy();
```

Comment:
@ShikharBhatt what do you think we should do with this test?

Reply:
For this one, not sure why am I getting undefined for the isComplete getter.

Reply:
Needs to be converted to instance. See https://github.com/impactility/vc-api/pull/30